package.json

```diff
@@ -15,7 +15,8 @@
     "@redhat-cloud-services/frontend-components": "2.4.9",
     "@redhat-cloud-services/frontend-components-notifications": "2.1.2",
     "@redhat-cloud-services/frontend-components-utilities": "2.1.1",
-    "@redhat-cloud-services/rbac-client": "^1.0.77",
+    "@redhat-cloud-services/rbac-client": "^1.0.78",
+    "axios-mock-adapter": "^1.18.1",
```

Comment:
Can you remove this dependency? We will not be using tests anymore. Instead, we will be mocking async requests via jest mock API.